### DISCUSSION TITLE
Add something about calendars to the xCDAT documentation

### DISCUSSION THREAD

**Discussion by jypeter** [xCDAT/xcdat#336] [3. Ideas]
@tomvothecoder We have ran into pandas time range limitations this morning ([Timestamp limitations](https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#timestamp-limitations) when using a _compliance checker script_ supplied by the ISMIP6 project

I have checked the xcdat site about this, and found that you have luckily already taken care of this problem. So has xarray

I have then checked what the xarray site was saying about the _exotic_ calendars our community uses (by [searching the site for noleap](https://docs.xarray.dev/en/v2022.06.0/search.html?q=noleap#), and it turns out that they don't say much more than [listing the supported calendars](https://docs.xarray.dev/en/v2022.06.0/user-guide/io.html?highlight=noleap#time-units)

Some **added-value of the xCDAT documentation** could be to **have a page covering the different calendars our community uses**, why we use them, and how to deal with them using xCDAT. I have [searched noleap on the xCDAT site](https://xcdat.readthedocs.io/en/latest/search.html?q=noleap) and you don't get much more information that _it is supported by `cftime`_.

There is [not much more information about _noleap_ on the cftime site](https://unidata.github.io/cftime/search.html?q=noleap). Basically, the only information is on the [4.4.1. Calendar](http://cfconventions.org/cf-conventions/cf-conventions#calendar) section of the CF convention website. This makes sense, but I'm not sure many users will go there, when (hopefully) using xCDAT as an intelligent wrapper around all this time/calendar _stuff_.

That's why a page about this in the xCDAT documentation would be nice (especially if people find it in Google).

The minimum information would be to have a **_Where do I get information about calendars_ question of the FAQ page**, leading to [4.4.1. Calendar](http://cfconventions.org/cf-conventions/cf-conventions#calendar)

**Comment by tomvothecoder** [comment]
Nice idea @jypeter. Including your suggestions, I'll think of ways to clearly convey calendar information.

**Comment by tomvothecoder** [comment]
This idea is addressed by #383!
